scripts/seedDB.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require("mongoose");
+const db = require("../models");
+
+// This file empties the Books collection and inserts the books below
```

Comment:
I'm going to assume this is copy pasted from another boilerplate and should be removed later